src/clj/rems/application/eraser.clj

```diff
@@ -10,14 +10,15 @@
 
 (defn- is-expired-draft?
   [application]
-  (let [expiration-threshold (-> env :application-expiration :draft Period/parse)
+  (let [expiration-threshold (some-> env :application-expiration :application.state/draft Period/parse)
         last-activity (:application/last-activity application)
         state (:application/state application)]
-    (and (= :application.state/draft state)
-         (time/before? last-activity (time/minus (time/now) expiration-threshold)))))
+    (when expiration-threshold
+      (and (= :application.state/draft state)
+           (time/before? last-activity (time/minus (time/now) expiration-threshold))))))
```

Comment:
This looks a bit strange when the `expiration-treshold` is not in the same `and`.